blingfire/Cargo.toml

```diff
@@ -1,7 +1,6 @@
 [package]
 name = "blingfire"
-version = "0.3.1"
-authors = ["reinfer Ltd. <eng@reinfer.io>"]
```

Comment:
I think authors is a required field for publishing to crates.io?

https://doc.rust-lang.org/cargo/reference/manifest.html#the-authors-field

Reply:
Ohh, I didn't mean to remove that, I had a merge conflict, well spotted!

Reply:
Fixed